### PR TITLE
GEOPY-1130: Prepare release las-geoh5 v0.1.0

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1130 - Prepare release las-geoh5 v0.1.0**

### CODE REVIEW

File: README.md
Comment (sebhmg) [inline README.md:1]: other repos are using rst, rather than md
(more common in the Python eco-system)

File: pyproject.toml [hunk 1]
```diff
@@ -1,16 +1,30 @@
 [tool.poetry]
 name = "las-geoh5"
-version = "0.0.1"
+version = "0.1.0"
 description = "Las/Geoh5 conversion"
 authors = ["Mira Geoscience <support@mirageoscience.com>"]
+keywords = ["geology", "geophysics", "earth sciences"]
```
Comment (sebhmg) [inline pyproject.toml:6]: must set the license type, and point to the README file

For license keyword to use for MIT, see https://python-poetry.org/docs/pyproject/#license

And no need to place the license in classifiers: poetry will take care of that

Comment (sebhmg) [reply pyproject.toml:6]: add homepage as well (see pyproject in geoh5py)

**Review by sebhmg — CHANGES_REQUESTED** [review]
see comment for a few missing things in pyproject.toml

File: pyproject.toml (hunk 1 above)
Comment (domfournier) [reply pyproject.toml:6]: and doner

File: pyproject.toml [hunk 2]
```diff
@@ -29,6 +43,8 @@ include_trailing_comma = true
 force_grid_wrap = 0
 use_parentheses = true
 line_length = 88
+# auto-updated by seed-isort-config
+known_third_party = ["geoh5py", "lasio", "numpy", "pytest", "toml", "tqdm"]
```
[the remark this answers is not in the document]
Comment (domfournier) [reply pyproject.toml:47]: Done

File: pyproject.toml [hunk 3]
```diff
@@ -2,8 +2,11 @@
 name = "las-geoh5"
 version = "0.1.0"
 description = "Las/Geoh5 conversion"
+license = "MIT"
+readme = "README.md"
+homepage = "https://mirageoscience.com"
 authors = ["Mira Geoscience <support@mirageoscience.com>"]
-keywords = ["geology", "geophysics", "earth sciences"]
+keywords = ["packaging", "poetry"]
```
Comment (sebhmg) [inline pyproject.toml:9]: previous keywords were goo, no?

Comment (domfournier) [reply pyproject.toml:9]: Thought you asked to change the keywords

**Review by sebhmg — APPROVED** [review]

**Pull request merged**